src/NLog.Database/DatabaseTarget.cs

```diff
@@ -78,7 +81,6 @@ namespace NLog.Targets
     /// <code lang="C#" source="examples/targets/Configuration API/Database/MSSQL/Example.cs" height="630" />
     /// </example>
     [Target("Database")]
-    [Target("DB")]
```

Comment:
Is this on purpose?

Reply:
Yes saw it while fixing the xml-docs, and prefer not to promote [acronyms](https://acronyms.thefreedictionary.com/DB)

Reply:
Its disappointing this in a bit hidden in a large pr. 

(Its not on the title, summary or even related)

Reply:
Ok. Will restore it, and remove it again in new PR.